rs/xnet/payload_builder/src/lib.rs

```diff
@@ -44,7 +45,7 @@ use ic_types::{
     xnet::{CertifiedStreamSlice, RejectSignal, StreamIndex},
     Height, NodeId, NumBytes, RegistryVersion, SubnetId,
 };
-use ic_xnet_hyper::{ExecuteOnRuntime, TlsConnector};
+use ic_xnet_hyper::TlsConnector;
```

Comment:
double check but I believe the TlsConnector is not used inside ic_xnet_hyper.

So you can move it to this file and finally completely separate the xnet client and server code.

Reply:
Or we can do this move in a follow-up MR as well.